### DISCUSSION TITLE
update to latest reference/vcp

### DISCUSSION THREAD

**Discussion by methionine23** [broadinstitute/seqr#3734] [General]
For local install, how to get the most recent reference and VEP download? When I tried download_reference_data and update_all_reference_data, VEP release 99 and genecode 39 were downloaded.

**Comment by hanars** [comment]
Those are the latest supported versions for seqr If you want to use a more recent version, you would need to update the seqr pipeline code directly to support more recent versions. We are hoping to release a supported update to a more recent version in 2024

**Reply by methionine23** [reply]
Great to know. Thanks!
